src/append/opentelemetry.rs

```diff
@@ -32,10 +34,7 @@ use opentelemetry_sdk::logs::LoggerProvider;
 use crate::append::Append;
 use crate::Layout;
 
-/// The communication protocol to opentelemetry that used when exporting data.
-///
-/// This is a logical re-exported [`Protocol`] to avoid version lock-in to
-/// `opentelemetry_otlp`.
+/// Specifies the wire protocol to use when sending logs to OpenTelemetry.
```

Comment:
```suggestion
/// Specifies the wire protocol to use when sending logs to OpenTelemetry.
///
/// This is a logical re-exported [`Protocol`] to avoid version lock-in to
/// `opentelemetry_otlp`.
```